### PR TITLE
Add Coin Change

### PR DESCRIPTION
**Pull request by ATREAY**

### CODE REVIEW

**Review by Sabine91 — COMMENTED** [review]
I have verified much of the code but i am not great with JS so i will need someone to look into that code.

**Review by shhossain — CHANGES_REQUESTED** [review]
Remove and change the requested.

**Comment by ATREAY** [comment]
> Remove and change the requested.
Can you please elaborate like what changes to I need to do in the file.

**Comment by ATREAY** [comment]
Any Updates???

File: Algorithms/Dynamic Programming/Coin Change/README.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+	}
+}
+
+```
+
+## JAVASCRIPT
+
+```javascript
+
+<script>
```
Comment (shhossain) [inline Algorithms/Dynamic Programming/Coin Change/README.md:346]:
```suggestion
```

remove this

File: Algorithms/Dynamic Programming/Coin Change/README.md [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+	return table[sum];
+}
+
+// Driver Function to test above function
+var coins = [1, 2, 3];
+var n = coins.length;
+var sum = 4;
+document.write(countWays(coins, n, sum));
+
+</script>
```
Comment (shhossain) [inline Algorithms/Dynamic Programming/Coin Change/README.md:380]:
```suggestion
```

and this.

File: Algorithms/Dynamic Programming/Coin Change/README.md [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+			table[j] += table[j-coins[i]];
+
+	return table[sum];
+}
+
+// Driver Function to test above function
+var coins = [1, 2, 3];
+var n = coins.length;
+var sum = 4;
+document.write(countWays(coins, n, sum));
```
Comment (shhossain) [inline Algorithms/Dynamic Programming/Coin Change/README.md:378]:
```suggestion
console.log(countWays(coins, n, sum));
```

Change this.

**Review by shhossain — CHANGES_REQUESTED** [review]
Remove and change. @ATREAY

**Comment by ATREAY** [comment]
> Remove and change. @ATREAY

Ok

**Review by shhossain — APPROVED** [review]
@Sabine91 . Done merge this.
@ATREAY Approved. Merging.

**Comment by ATREAY** [comment]
@Sabine91 @shhossain 
Ok Thank You.

**Comment by AmareshMuddebihal** [comment]
hi

**Pull request merged**